src/android/app/src/main/java/io/github/lime3ds/android/fragments/EmulationFragment.kt

```diff
@@ -412,6 +419,14 @@ class EmulationFragment : Fragment(), SurfaceHolder.Callback, Choreographer.Fram
         setInsets()
     }
 
+    private fun rotateScreen() {
+        if (resources.configuration.orientation == Configuration.ORIENTATION_PORTRAIT) {
+            (context as? EmulationActivity)?.requestedOrientation = ActivityInfo.SCREEN_ORIENTATION_LANDSCAPE
+        } else {
+            (context as? EmulationActivity)?.requestedOrientation = ActivityInfo.SCREEN_ORIENTATION_PORTRAIT
```

Comment:
here would be SCREEN_ORIENTATION_UNSPECIFIED for working autorotation, but won't work if you click rotate screen twice

Reply:
Perhaps a menu would be more apt for this instead of a toggle. Think the Landscape Screen Layout button